frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -31,10 +34,16 @@ protected BaseGraphTable(TableModel model) {
   @Override
   @UIEffect
   public Component prepareRenderer(TableCellRenderer renderer, int row, int column) {
+    if (getRowCount() != rowWidths.size()) {
+      rowWidths = List.fill(getRowCount(), 0);
```

Comment:
Hmm by assigning a fresh, zero-filled list you're losing the data which was already in the list... possibly you want to just crop or extend the list without losing the existing data? or if we don't care about the pre-existing data when `getRowCount() != rowWidths.size()`, then document why so

Reply:
No, I won't crop an existing list. This fresh list assignment is intentional. Documentation left.